functional-tests/fn_bridge_deposit_reclaim_drt_seen.py

```diff
@@ -0,0 +1,292 @@
+        addr = get_address(0)
+        refund_addr = get_address(1)
+        recovery_addr = get_recovery_address(0, bridge_pk)
+
+        # First let's see if the EVM side has no funds
+        # Make sure that the el_address has zero balance
+        balance = int(rethrpc.eth_getBalance(f"0x{el_address}"), 16)
+        assert balance == 0, "EVM balance is not zero"
+
+        # Also make sure that the all addresses have zero balance
+        btc_addr_balance = get_balance(
+            addr,
+            btc_url,
+            btc_user,
+            btc_password,
+        )
+        self.logger.debug(f"Address BTC balance (before DRT): {btc_addr_balance}")
+        assert btc_addr_balance == 0, "BTC balance is not zero"
+        btc_refund_balance = get_balance(
+            refund_addr,
+            btc_url,
+            btc_user,
+            btc_password,
+        )
+        self.logger.debug(f"Refund BTC balance (before DRT): {btc_refund_balance}")
+        assert btc_refund_balance == 0, "BTC balance is not zero"
+        btc_recovery_balance_post = get_balance_recovery(
+            recovery_addr,
+            bridge_pk,
+            btc_url,
+            btc_user,
+            btc_password,
+        )
+        self.logger.debug(f"DRT BTC balance (before DRT): {btc_recovery_balance_post}")
+        assert btc_recovery_balance_post == 0, "BTC balance is not zero"
+
+        # Generate Plenty of BTC to address for the DRT
+        self.logger.debug(f"Generating 102 blocks to address: {addr}")
+        btcrpc.proxy.generatetoaddress(102, addr)
+        self.logger.debug(f"Generated 102 blocks to address: {addr}")
+        time.sleep(0.5)
+
+        # Generate Plenty of BTC to recovery address for the take back path
+        self.logger.debug(f"Generating 102 blocks to recovery address: {recovery_addr}")
+        btcrpc.proxy.generatetoaddress(102, recovery_addr)
+        self.logger.debug(f"Generated 110 blocks to recovery address: {recovery_addr}")
+        time.sleep(0.5)
+
+        # Grab the balances again
+        btc_addr_balance = get_balance(
+            addr,
+            btc_url,
+            btc_user,
+            btc_password,
+        )
+        self.logger.debug(f"Address BTC balance (after generating blocks): {btc_addr_balance}")
+        btc_recovery_balance_post = get_balance_recovery(
+            recovery_addr,
+            bridge_pk,
+            btc_url,
+            btc_user,
+            btc_password,
+        )
+        self.logger.debug(f"DRT BTC balance (after generating blocks): {btc_recovery_balance_post}")
+
+        # DRT same block
+        txid_drt = self.make_drt(ctx, el_address, bridge_pk, maturity=0)
+        self.logger.debug(f"Deposit Request Transaction: {txid_drt}")
+        time.sleep(1)
+
+        # Kill the operator so the bridge does not process the DRT transaction
+        # NOTE: The operator is aware of the DRT transaction before stopping
+        bridge_operator.stop()
```

Comment:
Instead of stopping the bridge which may or may not cause the deposit transaction to be published, we can set the `--message-interval` flag when instantiating the bridge client. This will require changing the bridge factory to allow that flag to be passed. Setting the value of this flag to a high-enough value means that the nonces and signatures will be polled very slowly which would delay the signature aggregation and the Deposit Transaction would not be published. You can take the additional measure to stop the bridge operators too.

Reply:
Done in 923d4ffea9601a37f8e044ba953340e80176104a